_posts/2017/05/2017-05-04-windows-installer.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Plans for Windows Installer"
+date: 2017-05-04
+time: "16:00:00"
+category: ["Community","Tools","Teaching"]
+---
+
```

Comment:
Might be worth adding in something here about what the SWC needs are: a (preferably) command-line text editor for use (primarily) during the shell and git modules of SWC workshops that works across all platforms (Windows, macOS, *nix) and that is easy or transparent to install along with the other tools we ask learners to have before showing up.